src/main/kotlin/com/ampnet/walletservice/service/impl/WithdrawServiceImpl.kt

```diff
@@ -128,7 +128,7 @@ class WithdrawServiceImpl(
             DepositWithdrawType.USER -> {
                 if (withdraw.ownerUuid != user) {
                     throw InvalidRequestException(
-                        ErrorCode.USER_MISSING_PRIVILEGE, "Withdraw does not belong to this user"
+                        ErrorCode.USER_ROLE_INVALID, "Withdraw does not belong to this user"
```

Comment:
This is not a user role problem. Only owner can edit his withdraw, this rule doesn't depend on his role.